app/src/main/java/uk/gov/onelogin/signOut/ui/SignOutScreen.kt

```diff
@@ -40,8 +42,12 @@ fun SignOutScreen(
                 goBack()
             },
             onPrimary = {
-                viewModel.signOut(context)
-                goToSignIn()
+                try {
+                    viewModel.signOut(context)
+                    goToSignIn()
+                } catch (e: SignOutError) {
+                    goToSignOutError()
```

Comment:
We can remove the suppression of the swallowed exception if we just log the error here

Reply:
done 7371b4188da43dbf62772c5fc06dabb6e51b25ca